src/airflow/airqo_etl_utils/data_warehouse_utils.py

```diff
@@ -0,0 +1,160 @@
+import pandas as pd
+
+from airqo_etl_utils.airqo_api import AirQoApi
+from airqo_etl_utils.bigquery_api import BigQueryApi
+
+
+def query_hourly_measurements(
+    start_date_time: str, end_date_time: str, tenant=None
+) -> list:
+    biq_query_api = BigQueryApi()
+    columns = [
+        "time",
+        "site_id",
+        "device",
+        "device_number",
+        "latitude",
+        "longitude",
+        "pm2_5",
+        "s1_pm2_5",
```

Comment:
love the usage of the s1_pm_25 going forward and differentiating it from the pm2_5. @NoahNsimbe